onbuild/Dockerfile

```diff
@@ -0,0 +1,29 @@
+#
+# Dockerfile for Storm Mesos framework
+#
+FROM mesosphere/mesos:0.27.0-0.2.190.ubuntu1404
```

Comment:
@dsKarthick Yeah, I think this would eliminate the need for any separate folders, build-release.sh can just pass the MESOS_RELEASE via build-args.  We could actually cut Docker releases with different tags this way.